docs/how-to/service-catalog.md

```diff
@@ -5,122 +5,93 @@ sidebar_label: Service Catalog
-The viewer provides a clear interface for code inspection without requiring a local checkout.
-
-Find out more in our [Code View guide](/getting-started/view-generated-code/).
-
-![Amplication Service Catalog Filter](./assets/service-catalog/code-view.png)
-
-## Pending Changes
-
-The Pending Changes tab gives you a complete overview of all modifications across your services before they're committed. It tracks everything from entity modifications to plugin updates, allowing you to review and verify changes before generating new code. This staging area ensures you have full control over what gets included in your next build.
-
-![Amplication Service Catalog Pending Changes Tab](./assets/service-catalog/pending-changes.png)
-
-## Settings
-
-Settings provides centralized control over your project-level service settings.
-Learn more about available options in our [Service Settings guide](/how-to/api-admin-ui-settings/).
+## Grid Columns
+The grid displays comprehensive information about each service through the following columns:
+
+- **Type** - Displays the resource type with visual indicators: purple brackets icon for services, green icon for message brokers, and distinct icons for other resource types
+- **Name** - The name of your service
+- **Owner** - Shows the assigned team or individual member responsible for the service. Clicking this field takes you to settings where you can modify ownership
+- **Code Generator** - Indicates whether the service uses Node.js or .NET
+- **Repository** - Shows the connected Git repository. Can be modified through the service's "Sync with Git" tab
+- **Description** - Custom description of your service, editable through the service settings
+- **Pending Changes** - Displays the number of uncommitted changes since the last build
+- **Last Build** - Shows build status (green checkmark for success) and timing (e.g., "2 days ago"). Clicking navigates to the specific commit details
+- **Code Gen Version** - Displays the current code generator version with status indicators:
+  - Lightning bolt: automatically updates to latest version
+  - Lock icon: version is locked to specific release
+- **Template** - Shows if the service was created from a template (orange icon). Clicking navigates to template details
+- **Template Version** - Displays the current template version number, which increments with each template update
+- **Actions** - Provides access to available operations for the service
+
+## Customizing the Grid
+
+### Column Management
+
+- Drag columns left or right to reorder them
+- Resize columns by dragging their edges
+- Show/hide columns using the filter feature
+
+### Sorting
+
+Click any column header to sort the grid:
+
+- First click: Sort in ascending order
+- Second click: Sort in descending order
+- Third click: Remove sorting
+
+### Teams Organization
+
+Teams provide an additional layer of service organization:
```

Comment:
It also important to expose the owner of each service (for visibility, maintenance, technical issues or questions and such)